### ISSUE TITLE
Lex all the c keywords

### ISSUE THREAD

**Issue by kinggoesgaming** [onehr/crust#12]
Currently, we only detect only a few of them, but let's not discriminate :)

**Comment by onehr** [comment]
Need to worry about this: https://en.wikibooks.org/wiki/C_Programming/C_trigraph

**Comment by austinpgraham** [comment]
Do you mind assigning me to this issue? That way I'll be notified.

**Comment by onehr** [comment]
@austinpgraham Yes, I will, but your name can not show in the assignees list, so I will assign it to you later when I can see your name in the assignees list.

**Comment by onehr** [comment]
> 
> 
> Need to worry about this: https://en.wikibooks.org/wiki/C_Programming/C_trigraph

Trigraph should be converted in preprocessor.

**Comment by gitChang** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
FloydRose97, @tshrikant97, @botasylum, @DanielPhoton, @shashankG26, @falberto, @amebamcare, @aguasone, @xriz1992, @BigerXie, @andreyleskov, @johnteee, @Maydatr, @Loupou9, @jiaming7674, @KDunc11, @victorgrubio, @lenny-r, @renaldas, @kiddoneal
